mlos_bench/mlos_bench/tests/environments/local/__init__.py

```diff
@@ -30,32 +32,70 @@ def create_local_env(tunable_groups: TunableGroups, config: Dict[str, Any]) -> L
 
     Returns
     -------
-    local_env : LocalEnv
+    env : LocalEnv
         A new instance of the local environment.
     """
     return LocalEnv(name="TestLocalEnv", config=config, tunables=tunable_groups,
                     service=LocalExecService(parent=ConfigPersistenceService()))
 
 
-def check_local_env_success(local_env: LocalEnv,
-                            tunable_groups: TunableGroups,
-                            expected_results: Dict[str, float],
-                            expected_telemetry: List[Tuple[datetime, str, Any]]) -> None:
+def create_composite_local_env(tunable_groups: TunableGroups,
+                               params: Dict[str, Any],
+                               local_configs: List[Dict[str, Any]]) -> CompositeEnv:
+    """
+    Create a CompositeEnv with several LocalEnv instances.
+
+    Parameters
+    ----------
+    tunable_groups : TunableGroups
+        Tunable parameters (usually come from a fixture).
+    params: Dict[str, Any]
+        Additional config params for the CompositeEnv.
+    local_configs: List[Dict[str, Any]]
+        Configurations of the local environments.
+
+    Returns
+    -------
+    env : CompositeEnv
+        A new instance of the local environment.
+    """
+    return CompositeEnv(
+        name="TestCompositeEnv",
+        config={
+            **params,
+            "children": [
+                {
+                    "name": f"TestLocalEnv{i}",
+                    "class": "mlos_bench.environments.local.local_env.LocalEnv",
+                    "config": config,
+                }
+                for (i, config) in enumerate(local_configs)
+            ]
+        },
+        tunables=tunable_groups,
+        service=LocalExecService(parent=ConfigPersistenceService()),
+    )
+
+
+def check_env_success(env: Environment,
```

Comment:
was: `check_local_env_success`, but the input env does not have to be `LocalEnv`. Hence the rename